src/App.re

```diff
@@ -134,6 +135,10 @@ let init = (solver, spec, config, meta, cil, analyses, warnings, stats, file_loc
     | _ => raise(InitFailed("Failed to load file path table"))
     };
 
+  let cilfilename = "file.cil";
+  let oc = Stdlib.open_out(cilfilename);
+  dumpFile(defaultCilPrinter, oc, cilfilename, cil);
```

Comment:
It should be possible to use `printGlobal` (per-global) to directly invoke `defaultCilPrinter` to produce a `Pretty.doc` which can be converted to string with `Pretty.sprint`.

But this is probably simpler for now.